ci/test.groovy

```diff
@@ -192,7 +192,7 @@ nm.slack_notify() {
         archiveArtifacts(artifacts: '*.tgz')
         archiveArtifacts(artifacts: '*.md5')
         archiveArtifacts(artifacts: '*.txt')
-        stash name: 'text_files', includes: '*.txt'
+        stash name: 'text_files', includes: '*.txt', excludes: '*ReleaseNotes.txt'
```

Comment:
Do we care that this will still result in it being included in the `*tarball-doxbox.zip`?

Reply:
👍 Wasn't sure if we cared about the tarball or not. I'll update it

Reply:
Oh I see, sorry I hadn't clocked that was the only one of the three affect. Probably worth getting right to retain the option.